CHANGES.md

```diff
@@ -77,6 +71,15 @@ Various enhancements to `cylc lint`:
 
 ### Fixes
 
+[#5525](https://github.com/cylc/cylc-flow/pull/5525) - Jobs can use scripts
+in `share/bin` and Python modules in `share/lib/python`.
```

Comment:
Wasn't this correct before? This an enhancement rather than a fix IMO

Reply:
Merge err on my part :(